### PR TITLE
Added game setting update, some minor design fix

### PR DESCRIPTION
**Pull request by hiroya210**
Host can now update game settings:

- I added "max_rounds" column to Instance model
- Implemented broadcast_to when game settings updated to reflect to all players without refreshing
- Flash, to notify the update to everyone on the instance
- Some minor design fix on modals

https://user-images.githubusercontent.com/79787121/155245075-b485a475-3938-4639-adbb-da5d1ee99e93.mp4

### CODE REVIEW

**Comment by spioupiou** [comment]
Looks awesome @hiroya210! I was just wondering, should we really allow them to change the pin code? There is a chance that other unrelated players might be playing at the same time, if both groups of friends chose 000000 as their pin, we might run into some troubles...

**Comment by carlnoval** [comment]
As discussed, should be fine to allow pins to be updated for now. We'll disable it on another PR.

File: app/controllers/instances_controller.rb [hunk 1]
```diff
@@ -7,8 +7,8 @@ def create
     @instance = Instance.create!(
       game_id: params[:game_id],
       user_id: current_or_guest_user.id
```
Comment (carlnoval) [inline app/controllers/instances_controller.rb:9]: @hiroya210 not quite done yet but the missing comma here is throwing an error.

**Comment by carlnoval** [comment]
OK this is good, played with it locally :)

Additional task in the future [trello](https://trello.com/c/5BOaiVsQ/20-implement-max-rounds-show-only-end-game-at-last-round-take-players-back-to-new-dbi-instance).

**Pull request merged**